scripts/pi-hole/js/index.js

```diff
@@ -139,14 +139,12 @@ function updateQueriesOverTime() {
         // convert received objects to arrays
         data.domains_over_time = objectToArray(data.domains_over_time);
         data.ads_over_time = objectToArray(data.ads_over_time);
-        // remove last data point since it not representative
-        data.ads_over_time[0].splice(-1,1);
```

Comment:
This is a change in functionality, is the comment out of date (the last point is representative)?

Reply:
It depends on your point of view. The last point is the *current* overTime interval, i.e., it is 09:30-09:40 if your local time is 09:31. This interval is still filling up so the curve may go down there at 09:31 and may go up at 09:39. It is still somehow representative as it is the true number of queries we've seen until now.

Reply:
I can separate this change into another PR in case we want to discuss this more.

Reply:
I don't think users expect the points of the graph to change their value. Most of the time, the last data point will be less than the previous amount and can't be compared to the rest of the graph because it does not cover the same amount of time.